src/App.js

```diff
@@ -1,5 +1,37 @@
+import CarsModel from "./CarsModel.js";
+import { ERROR_MESSAGES, MESSAGES } from "./constants/message.js";
+import {
+  getInputAsync,
+  printCarsMove,
+  printWinners,
+} from "./interfaceUtils.js";
+import { isCarNamesValid, isTrialInputValid } from "./validation.js";
+
 class App {
-  async run() {}
+  async run() {
+    const carsUserInput = await getInputAsync(MESSAGES.INPUT_CAR_NAMES);
+    const carsModel = new CarsModel(carsUserInput);
+
+    const carNames = carsModel.getCarNames();
+    const { isCarValid, errCarMessage } = isCarNamesValid(carNames);
```

Comment:
```suggestion
    const { isCarValid, errorCarMessage } = isCarNamesValid(carNames);
```
저는 축약하지 않는 편인 것 같은데, 지예님은 축약을 사용하신 것 같네요!

축약하실 때 지예님만의 기준이 있으신가요?? 

사실 에러가 `err`로 축약되어 있어서 뒤에 메세지도 `msg`로 축약되어 있을 줄 알았는데 에러만 축약되어 있어서 기준이 더 궁금해지는 것 같네요🤔

Reply:
아... `errCarMsg` 하려다가.. 이건 너무 줄였나? 싶었어요 ㅎㅎ 별로 중요하지도 않은 Car만 돋보이는 느낌이었구요... 그래서 `err`와 `Msg` 중에서는 어쨌든 '메시지'라는게 중요하니까 `Message`만 축약하지 않았던 것 같아요!

제가 고민했던 부분을 정확하게 집어주시다니.. 
![image](https://github.com/user-attachments/assets/dc162b9a-a8e0-47ee-879a-7f5268da5fb4)
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 감사합니당 👍